src/util/send-notification.ts

```diff
@@ -0,0 +1,74 @@
+import * as notifications from '../notifications/notifications'
+import createNotif from './notifications'
+import checkIfOnline from './check-network'
+import { remoteFunction } from './webextensionRPC'
+import { browser } from 'webextension-polyfill-ts'
+
+const storeNotification = remoteFunction('storeNotification')
+const estimateBackupSize = remoteFunction('estimateInitialBackupSize')
+const getDriveSize = remoteFunction('getDriveSize')
+
+async function sendNotification(id: string) {
```

Comment:
I'm confused about the names of the two main functions. They use the verbs "send" and "dispatch", although `sendNotification` eventually calls `dispatchNotification`. 

Also the string literals, like `'error'`, `'backup_error'`, `'drive_size_empty'`, etc. should probably be declared as constants somewhere and re-used rather than rewriting them every time. How about in `src/notifications/constants` module?